qa/L0_trace/test.sh

```diff
@@ -806,6 +833,18 @@ fi
 if [ `grep -c "\"trace_count\":\"-1\"" ./curl.out` != "1" ]; then
     RET=1
 fi
+if [ `grep -c "\"trace_mode\":\"OPENTELEMETRY\"" ./curl.out` != "1" ]; then
+    RET=1
+fi
+if [ `grep -c "\"bsp_max_export_batch_size\":\"512\"" ./curl.out` != "1" ]; then
+    RET=1
+fi
+if [ `grep -c "\"bsp_schedule_delay\":\"5000\"" ./curl.out` != "1" ]; then
+    RET=1
+fi
+if [ `grep -c "\"bsp_max_queue_size\":\"2048\"" ./curl.out` != "1" ]; then
+    RET=1
+fi
```

Comment:
We might also check that settings do not contain `trace file` and `log frequency`. Same for trace mode triton : no OpenTelemetry - related settings